tests/configlet/util/generic_patch.py

```diff
@@ -0,0 +1,217 @@
+#! /usr/bin/env python
+
+import argparse
+import json
+import jsonpatch
+import fnmatch
+import os
+import re
+import sys
+import time
+import yaml
+
+from helpers import *
+from common import *
+
+if os.path.exists("/etc/sonic/sonic-environment"):
+    from mock_for_switch import config_reload, wait_until
+else:
+    from tests.common.config_reload import config_reload
+    from tests.common.utilities import wait_until
+
+CMD_APPLY_HACK = "config apply-patch -n -i '' -v {}"
+CMD_APPLY = "config apply-patch -n -i /FEATURE -i /SCHEDULER -i /QUEUE -i /CABLE_LENGTH -i /ACL_TABLE/E -v {}"
```

Comment:
curious: in`/ACL_TABLE/E` what does `E` mean? I don't think there is an ACL table named `E`

Reply:
Mistake 
;)
Good catch